public/scripts/editor/js/fc/bramble-menus.js

```diff
@@ -71,20 +77,59 @@ define(function(require) {
       bramble.addNewFile(type);
     }
 
-    $("#filetree-pane-nav-add-html").click(function() {
+    function downloadFileToFilesystem(location, localPath, cb) {
+      cb = cb || function noop() {};
+
+      return $.get(location)
+      .then(function(data) {
+        bramble.addNewFileWithContents(localPath, data, function(err) {
+          if (err) {
+            console.error("[Bramble] Failed to write " + localPath, err);
+          }
+          cb();
+        });
+      }, function(err) {
+        if (err) {
+          console.error("[Bramble] Failed to download " + location, err);
+        }
+        cb();
+      });
+    }
+
+    $addHtml.click(function() {
       addFileType("html");
     });
-    $("#filetree-pane-nav-add-css").click(function() {
+    $addCss.click(function() {
       addFileType("css");
     });
-    $("#filetree-pane-nav-add-js").click(function() {
+    $addJs.click(function() {
       addFileType("js");
     });
+    $addTutorial.click(function() {
+      // TODO: We should probably add a loading indicator here
+      downloadFileToFilesystem('/tutorial/tutorial.html', '/tutorial.html', function() {
+        // TODO: The load is finished here
+        menu.close();
```

Comment:
I wonder if you should close the menu as soon as the click happens, so it looks like something happened, and they don't keep clicking while it downloads.